Sources/Collections/ObservableArray.swift

```diff
@@ -253,6 +253,10 @@ extension ObservableArray: DataSourceProtocol {
   public func numberOfItems(inSection section: Int) -> Int {
     return count
   }
+
+  public func item<Item>(at index: Int) -> Item {
```

Comment:
Ahhh yes, that did it. SourceKitService.got it's pants caught around it's ankles again. Thanks for the advice!